torch/fx/subgraph_rewriter.py

```diff
@@ -1,21 +1,34 @@
+from dataclasses import dataclass, field
 from .graph_module import GraphModule
 from .graph import Graph
 from .node import Node
 from ._symbolic_trace import symbolic_trace
 from ._compatibility import compatibility
 
 import copy
-from typing import Callable, Dict, List, NamedTuple, Optional, Set
+from typing import Callable, Dict, List, Optional, Set
 import torch
 
 __all__ = ['Match', 'replace_pattern']
 
-@compatibility(is_backward_compatible=True)
-class Match(NamedTuple):
+@compatibility(is_backward_compatible=False)
```

Comment:
"did you get clearance for this" lol

Reply:
No... 
but this class really shouldn't be backward compatible.